content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -2,35 +2,82 @@
-  linked by default. Open its *External link* to enable the feature *Bill from tasks*.
+To turn on the :guilabel:`Timesheet on Ticket` feature, go to :menuselection:`Helpdesk -->
+Configuration --> Helpdesk Teams`, select a Helpdesk team, and then, click :guilabel:`Edit`. Next,
+enable the options :guilabel:`Timesheet on Ticket` and :guilabel:`Time Reinvoicing`.
+
+In the :guilabel:`Timesheet on Ticket` drop-down option, choose the Project to which tickets (and
+timesheets) will be linked by default. Then, open the Project's external link to edit its settings
+and enable the :guilabel:`Billable` option. Finally, save both the Project and the Helpdesk team's
+settings to apply these features.
 
-.. image:: media/reinvoice_time1.png
+.. image:: reinvoice_from_project/project_billable.png
    :align: center
-   :alt: Bill from Tasks in Odoo Helpdesk
+   :alt: Billable feature in Project settings.
+
+Create a sales order
+====================
 
-Create a sales order and an invoice
-====================================
+After a ticket comes into the pipeline, go to :menuselection:`Sales --> Create`. Then, add the
+customer from the ticket to the new quotation. In the :guilabel:`Order Lines` tab, add the
+:guilabel:`Service on Timesheet` product. Finally, click :guilabel:`Confirm` to turn the quotation
+into a sales order.
 
-Now, once you have recorded the time you spent on the helpdesk ticket, under the *Timesheets* tab,
-access the task clicking on its name.
+.. tip::
+   Remember the :abbr:`SO (Sales Order)` name for later to link the Helpdesk ticket to the correct
+   sales order.
 
-.. image:: media/reinvoice_time2.png
+.. image:: reinvoice_from_project/service_on_timesheet.png
    :align: center
-   :alt: Sales Order from a task in Odoo Helpdesk
+   :alt: Add Service on Timesheet product to the SO.
 
-*Create Sales Order* and proceed to create the invoice.
+Record a timesheet
+==================
 
-.. image:: media/reinvoice_time3.png
+To record a timesheet, jump back to the Helpdesk ticket by going to the Helpdesk dashboard and
+clicking :guilabel:`Tickets` on the team's card (or by going to the Projects dashboard and clicking
+:guilabel:`Tickets` on the project card). Then, click :guilabel:`Edit` and use the
+:guilabel:`Timesheets` tab to record the time spent on the ticket.
+
+.. image:: reinvoice_from_project/record_timesheet_ticket.png
    :align: center
-   :height: 300
-   :alt: Sales Order from a task in Odoo Helpdesk
+   :alt: Record time spent on a ticket.
+
+Link the Helpdesk ticket to the SO
+==================================
+
+To link the :abbr:`SO (Sales Order)` to the ticket, start on the ticket form. Then, click
+:guilabel:`Edit` and click the :guilabel:`Sales Order Item` field. Next, select the :abbr:`SO
+(Sales Order)` that was created earlier from the drop-down menu. Lastly, click :guilabel:`Save` to
+connect the ticket and the :abbr:`SO (Sales Order)`.
```

Comment:
Suggestion.
First sentence may have been ended too early before an explicit/actionable instruction was given.
Helps the flow/rhythm of the paragraph too imo...

Up to you though as I'm being nitpicky at this point 😉 

```suggestion
To link the :abbr:`SO (Sales Order)` to the ticket, start on the ticket form, click
:guilabel:`Edit` and then click the :guilabel:`Sales Order Item` field. Next, find and select the :abbr:`SO
(Sales Order)` that was created earlier from the drop-down menu. Lastly, click :guilabel:`Save` to
connect the ticket and the :abbr:`SO (Sales Order)`.
```